cmd/lakectl/cmd/ingest.go

```diff
@@ -37,8 +37,9 @@ func stageWorker(ctx context.Context, client api.ClientWithResponsesInterface, w
 }
 
 var ingestCmd = &cobra.Command{
-	Use:   "ingest --from <object store URI> --to <lakeFS path URI> [--dry-run]",
-	Short: "Ingest objects from an external source into a lakeFS branch (without actually copying them)",
+	Deprecated: "use import command instead",
+	Use:        "ingest --from <object store URI> --to <lakeFS path URI> [--dry-run]",
```

Comment:
Why do we explain how to use it if it's deprecated?

Reply:
@idanovo thanks - though it's deprecated, it is still usable. Do you think we should remove it?

Reply:
Ha, I thought it's not usable anymore